src/extensions/AyameMessage.js

```diff
@@ -95,6 +95,12 @@ module.exports = Structures.extend("Message", (Message) => {
     reply(content, options) {
       return this.send(APIMessage.transformOptions(content, options, { reply: this.member || this.author }));
     }
+
+    get locale() {
+      // TODO: Get guild settings and pass the locale here!
+      return this.client.locales.get("en-US");
```

Comment:
Ideally, you would want to get the guild settings and if they are not set, go to a default, like a this.client.options.defaultLocale property.